playbooks/roles/os_temps/tasks/main.yml

```diff
@@ -41,11 +41,17 @@
 - name: "Create project {{ openshift_project }}"
   import_tasks: get_set_project.yml
 
-# Import_tasks jq for querying container config files
+# Check if jq is already installed so we can skip installation
+- name: "Check if jq is already installed"
+  command: jq --version
+  register: jq_installed
+  ignore_errors: yes
```

Comment:
maybe add - failed_when: jq_installed>1

Reply:
I didn't want this task to ever fail, just to try running jq

Reply:
ack, then we are good, I just did not know if this was a hard req.